registry/cloudflare-workers/index.js

```diff
@@ -1,69 +1,170 @@
 const fetch = require('node-fetch')
 const fs = require('fs')
 
-const doWorkerDeploy = async ({ zone_id, script_path }) => {
+const removeWorker = async ({ account_id, script_name }, context) => {
   const AUTH_KEY = process.env.CLOUDFLARE_AUTH_KEY
   const EMAIL = process.env.CLOUDFLARE_EMAIL
-  const url = `https://api.cloudflare.com/client/v4/zones/${zone_id}/workers/script`
+
+  const url = `https://api.cloudflare.com/client/v4/accounts/${account_id}/workers/scripts/${script_name}`
 
   const headers = {
     'X-Auth-Email': `${EMAIL}`,
     'X-Auth-Key': `${AUTH_KEY}`,
-    Accept: 'application/javascript',
-    'Content-Type': 'text/javascript'
+    'Content-Type': 'application/javascript'
+  }
+
+  const response = await fetch(url, { headers, method: `DELETE` })
+  if (response.status == 200) {
+    context.log(`✅ Script Removed: ${response.url}`)
+  } else {
+    context.log(`❌ Script Remove Failed `)
+  }
+  return response
+}
+
+const removeRoute = async ({ zone_id }, context) => {
+  const AUTH_KEY = process.env.CLOUDFLARE_AUTH_KEY
```

Comment:
@pinkerton document that env vars are needed

Reply:
Thanks, will do. I also really like the emoji here 👍